### DISCUSSION TITLE
'cannot bind to path' Ansible folder permission issue

### DISCUSSION THREAD

**Discussion by Zaspath** [Sequel-Ace/Sequel-Ace#1249] [Q&A]
Hi,

I've migrated my DB connections over to Sequel Ace, and found I'm running into a permissions issue I'm not sure how to resolve, we use Ansible to and conect via a jumpbox:

`unix_listener: cannot bind to path /Users/<USER>/.ansible/cp/ansible-ssh-<SERVER>-22-<USER>.WiFyUTSb4LxETPGr: Operation not permitted`

The path doesn't exist, so I added '/Users/<USER>/.ansible/cp/' to the access list but this didn't help, are there any ideas on how I can resolve this?

Please note, I've censored the user and server names above, in case it wasn't obvious :)

Thanks.

**Comment by Jason-Morcos** [comment]
Hi, my hunch is this has something to do with Sequel Ace being sandboxed whereas Sequel Pro was not. You could try granting Ace full disk access, but I'm not sure that will fix your issue. The faq on our website may also be useful! https://sequel-ace.com

**Comment by Zaspath** [comment]
Hi Jason, sadly, that resulted in the same outcome. It doesn't appear possible to add a folder or sub-folders, as I think that would probably solve it in the Accessible Files feature. Not sure there's much I can do then, unless this is considered in a future product update.
